woocommerce/woocommerce-android
# Fix feature feedback tag tracking

Summary
==========
Fixes issue #6118 by refactoring how we store the state of Feedback using the feature as the state key, this way, we make the feedback state related to the feature itself, not the requesting view, and when a new Feature is added, a new state is stored.

⚠️ With this change, we will end up resetting the state of each Feedback banner interaction with all merchants, but since the banner is non-intrusive and this bug affects our capability of displaying the banner for new relevant features, I think it's worth it.

How to Test
==========
1. Do a fresh install of the Woo app
2. Go to the Order list and hit `Dismiss` to the current feedback feature banner there
3. Go back to the code and add to the `FeedbackFeatureSettings` a new kind of Feature and change it inside the `ProductListFragment`or any other view using Feedback banners.
4. Verify that this new feature state is still set as `Unanswered` and the banner is correctly displayed for that view.

Update release notes:
